### DISCUSSION TITLE
Feedback for Dependabot on GHES!

### DISCUSSION THREAD

**Discussion by exvuma** [community/community#18004] [Enterprise]
Dependabot went [GA in GHES 3.5](https://docs.github.com/enterprise-server@3.5/admin/release-notes#dependabot-updates-are-generally-available). After setting up Dependabot updates, you'll now be able to see Dependabot jobs run as actions:
<img width="440" alt="Screen Shot 2022-06-15 at 5 22 25 PM" src="https://user-images.githubusercontent.com/7578652/173942112-74d2ded7-eb5a-4529-970b-7cf69cd4d352.png">
 Would you like to see Dependabot run as an Action for repo’s hosted on gitub.com 🧐  
 We'd love to hear what you think. Please share your feedback here.

**Comment by Wilhelmsson177** [comment]
I enabled Dependabot on GHES, but I have an issue with a self-signed certificate. How do I add the certificate to the used or is it possible to build my own dependabot-core image, if so how do I change the used image.

![image](https://user-images.githubusercontent.com/16141053/179224278-dc0efe1f-929e-442b-9cab-191331ab8f16.png)

**Reply by landongrindheim** [reply]
👋 @Wilhelmsson177 Thanks for the question! Please forgive our very late response. If you're still experiencing this issue, could you try [adding the certificate to the runner](https://docs.github.com/en/enterprise-server@3.5/admin/github-actions/advanced-configuration-and-troubleshooting/troubleshooting-github-actions-for-your-enterprise#configuring-self-hosted-runners-when-using-a-self-signed-certificate-for-github-enterprise-server) using the environment variable `CUSTOM_CA_CERT`?

As of GHES 3.6, Dependabot will be able to pick up the variable (`NODE_EXTRA_CA_CERTS`) mentioned in the [Action docs](https://docs.github.com/en/enterprise-server@3.5/admin/github-actions/advanced-configuration-and-troubleshooting/troubleshooting-github-actions-for-your-enterprise#configuring-self-hosted-runners-when-using-a-self-signed-certificate-for-github-enterprise-server). In the meantime, `CUSTOM_CA_CERTS` _should_ do the trick 🙂

**Reply by Wilhelmsson177** [reply]
Hey @landongrindheim,

I apologize for the late answer. The notification must have been missed from my side.

I tried setting both environment variables on the runner, but it did not work.

I tried the .env file approach and I now receive a new error. Kinda success, :-) The error is 
```
Error: Dependabot encountered an unexpected problem

Error: Request failed with status code 400
```

If I follow the link, I see the Dependency graph, but there are no logs.
